lando/k8s/jobmanager.py

```diff
@@ -279,6 +351,8 @@ def __init__(self, job):
         self.workflow_path = '{}/{}'.format(Paths.WORKFLOW, os.path.basename(job.workflow.url))
         self.job_order_path = '{}/job-order.json'.format(Paths.JOB_DATA)
         self.system_data = 'system-data-{}'.format(suffix)
+        self.run_workflow_stdout_path = '{}/cwltool-output.json'.format(Paths.OUTPUT_DATA)
```

Comment:
Is there a reason to keep using the `cwltool-output.*` names? We're making these changes to use a different implementation than cwltool. Are there other parts of Bespin that expect these names?

For a generic name, I'd suggest `bespin-workflow-output.json` and `bespin-workflow.log` since `cwltool` implies a specific implementation and doesn't jump out as where to look unless you know that `cwltool` is the process that ran your analysis (and in this case it isn't 😄 )

It might be more informative to name these after the job id, job name, or workflow name/tag. Probably an improvement issue for that.